### ISSUE TITLE
Dates don't work as arguments in uislider and uirange

### ISSUE THREAD

**Issue by jchojna** [Appsilon/shiny.semantic#227]
Some quick test:
  
works in shiny function
```
sliderInput("shiny_slider",
              "Shiny Slider",
              as.Date("2019-01-23"),
              as.Date("2019-03-23"),
              as.Date("2019-01-23")),
```
not working in semantic

```
uislider("semantic_slider",
           value = as.Date("2019-01-23"),
           min = as.Date("2019-01-23"),
           max = as.Date("2019-03-23")),
```

Similarly in uirange

**Comment by dokato** [comment]
related to #160

**Comment by rabiibouhestine** [comment]
Fomantic UI slider does not seem to support dates out of the box ([docs](https://fomantic-ui.com/modules/slider.html)).

One solution is to add a labels parameter to `shiny.semantic::sliderInput` then use fomantic `interpretLabel` function to assign a date as label to each value in the slider, but I believe this is not optimal as we will have to define a list of dates then we will have to work with the dates indexes in the list.

Maybe leave dates to be handled by `calendar` instead?  We can even add a `calender_range` wrapper for the [fomantic calender range](https://fomantic-ui.com/modules/calendar.html#range) ? @dokato ?

**Comment by dokato** [comment]
Agreed, date range would be super useful, and that difficult to do with the calendar we have(?).

**Comment by rabiibouhestine** [comment]
We can just use the `calendar()` that we have and define two calendars for a start date and an end date, implementing a `calender_range()` will only allow us to put two calendars in one single widget basically.

**Comment by dokato** [comment]
Yes, exactly, something like that. But also coming back to the original topis, if shiny slider supports the dates, we should at least throw a warning that this feature is not implemented in Fomantic.

**Comment by esraay** [comment]
This issue will be fixed with #363
